server/config.ts

```diff
@@ -54,7 +54,7 @@ export default {
   gitRef: get('GIT_REF', 'xxxxxxxxxxxxxxxxxxx', requiredInProduction),
   branchName: get('GIT_BRANCH', 'xxxxxxxxxxxxxxxxxxx', requiredInProduction),
   production,
-  https: production,
+  https: process.env.NO_HTTPS === 'true' ? false : production,
```

Comment:
This was preventing log-in as it couldn't create the session cookie before redirecting to auth. 
When Auth redirected back the application refused to accept the request as it couldn't validate state.

Reply:
it was a right pain to track down 😅